scanpy/get/__init__.py

```diff
@@ -3,3 +3,5 @@
 
 # Private
 from .get import _get_obs_rep, _set_obs_rep
+
+from .aggregated import aggregated, aggregated_from_array
```

Comment:
```suggestion
from ._aggregated import aggregated
```

* I am curious about how it works to overwrite the field in the namespace
* I don't think we need to export the `aggregated`